vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/model/SwipeCoordinates.java

```diff
@@ -38,4 +40,25 @@ public Point getEnd()
     {
         return end;
     }
+
+    @Override
+    public int hashCode()
+    {
+        return Objects.hash(end, start);
+    }
+
+    @Override
+    public boolean equals(Object obj)
```

Comment:
why is it needed?

Reply:
for unit tests :)

Reply:
The code for unit tests should live in unit tests

Reply:
removed as no more needed due api changes